apidoc/Titanium/UI/WebView.yml

```diff
@@ -452,6 +452,13 @@ properties:
     default: false
     platforms: [iphone, ipad]
 
+  - name: blacklistFirstLoad
+    summary: Determines if the initial webview url will be checked against the blacklist.
+    platforms: [iphone,iphone]
+    type: Boolean
+    default: false
+    since: "5.4.0"
```

Comment:
As discussed, I don't like that properly. As it also causes parity problems, we should remove it and check every URL that is in the blacklist. If they want to exclude certain domains, they should remove them from the blacklist or set the `blacklistedURLs` after its creation, which should be possible.